Godeps/Godeps.json

```diff
@@ -34,8 +33,8 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runtime-spec/specs-go",
-			"Comment": "v1.0.0-rc1-31-gbb6925e",
-			"Rev": "bb6925ea99f0e366a3f7d1c975f6577475ca25f0"
+			"Comment": "v1.0.0-rc5-30-g3adac26",
```

Comment:
Do you mean to only update to `v1.0.0-rc5`? Or like https://github.com/opencontainers/runtime-tools/pull/349/commits/237bca691ca6d2179701322961a67b34873b3ab8?